envoy.code.check/envoy/code/check/typing.py

```diff
@@ -17,6 +17,7 @@ class BaseExtensionMetadataDict(TypedDict):
 
 class ExtensionMetadataDict(BaseExtensionMetadataDict, total=False):
     undocumented: bool
+    type_urls: Optional[List[str]]
```

Comment:
no need for `Optional` as the `TypedDict` is set to `total=False`

Reply:
Thanks. This is all new to me in python.

Reply:
yep, its fairly new to python